Assets/IntegrationTests/TestShopifyCollections.cs

```diff
@@ -36,5 +37,41 @@ public class TestShopifyCollections : MonoBehaviour {
 
             Assert.IsTrue (waiter.IsStopped, Utils.MaxQueryMessage);
         }
+
+        [UnityTest]
+        public IEnumerator LoadAllCollections() {
+            float maxDuration = 6f;
+            int maxCollections = 13;
+            StoppableWaitForTime waiter = new StoppableWaitForTime (maxDuration);
+            ShopifyError errorsFromQueries = null;
+            List<Collection> AllCollections = new List<Collection>();
+
+            Action<string> loadPage = null;
+
+            loadPage = (string after) => {
```

Comment:
This can be condensed to `var loadPage = (...) => ...`

Reply:
Unfortunately it can't because it's a recursive lambda if you don't drop it to two lines it means that `loadPage` doesn't exist at the time of declaration.

Reply:
ahh, right